rs/canister_sandbox/src/replica_controller/sandbox_process_eviction.rs

```diff
@@ -26,30 +28,29 @@ pub(crate) struct EvictionCandidate {
 /// 4. Return the evicted candidates.
 pub(crate) fn evict(
     mut candidates: Vec<EvictionCandidate>,
-    min_count_threshold: usize,
+    mut total_rss: NumBytes,
```

Comment:
Very minor point: would it make more sense to not accept a mutable `total_rss` here but rather that the function maintains a "running" total rss while it's evicting? It seems somewhat cleaner to me but I'm also ok with this.